yarn.lock

```diff
@@ -5030,18 +5030,16 @@
   resolved "https://registry.yarnpkg.com/@uniswap/token-lists/-/token-lists-1.0.0-beta.21.tgz#5a4844236e328d3b0e87161c19edaace669e3e6e"
   integrity sha512-IQ1YpWKgXIsfYqym9Nl/uaB1t5Wk70pyqJ2W4FmvQmt3FIQz+pHlzlEl91aYloPrlcZaFluE0Jq7gkfYpIjsBw==
 
-"@unstoppabledomains/resolution@3.0.0":
-  version "3.0.0"
-  resolved "https://registry.yarnpkg.com/@unstoppabledomains/resolution/-/resolution-3.0.0.tgz#148080757f3ab2e94993228a5934ea60f1e0807b"
-  integrity sha512-iAvVkDWsSdi9WOiCp8t6GwWQZYIQjca+JAzfnNB7LvRDim3jrgcLlbeSYu+Mwzn2JKUGf/Ygw7YtnmjihnT5fw==
+"@unstoppabledomains/resolution@8.3.1":
```

Comment:
So, here the tests are failing because the stealth keys are added in the StealthKeyRegistry of Umbra not Unstoppable Domains. 
Should I remove these tests or change the test to use StealthKeyRegistry of Umbra or any other solution you have?

Reply:
Ah, sorry, yes storing the keys directly on the name is now deprecated. We used to do that for both ENS and CNS, but for various reasons we migrated everything to the `StealthKeyRegistry`. So now we take an ENS/CNS name, resolve it to an address, and look up that address in the registry contract.

For the tests, we hardcode a name (previously it was `udtestdev-msolomon.crypto`, but we may need a new one now?) which has its pubkeys set on Rinkeby, and run the test against rinkeby (it's read-only, no txs are sent). We also are not forced to use rinkeby, since it's being deprecated anyway, so feel free to change those to a different network if you need—thank you!

Reply:
pubkeys set in Unstoppable Domains Resolver or Umbra StealthKeyRegistry?

Reply:
pubkeys should now always be set in the Umbra StealthKeyRegistry. It uses compressed keys, so the easiest way to do this is through the UI